readme.md

```diff
@@ -98,16 +98,138 @@ Tooltips will now also load in from fixtures:
 * You can add tooltips to your fixture by adding a Tooltips.txt on [FixtureRoot]/[FixtureName]_tooltips/Tooltips.txt
 * You also need to add two executions to your maven dependency plugin to be able to load fixture tooltips:
 
-                    <execution>
-                        <id>unpack-dependencies</id>
-                        <phase>compile</phase>
-                        <goals>
-                            <goal>unpack-dependencies</goal>
-                        </goals>
-                        <configuration>
-                            <includes>**/Tooltips.txt</includes>
-                            <outputDirectory>${project.basedir}/wiki/TooltipData</outputDirectory>
-                        </configuration>
-                    </execution>
+        <execution>
+            <id>unpack-dependencies</id>
+            <phase>compile</phase>
+            <goals>
+                <goal>unpack-dependencies</goal>
+            </goals>
+            <configuration>
+                <includes>**/Tooltips.txt</includes>
+                <outputDirectory>${project.basedir}/wiki/TooltipData</outputDirectory>
+            </configuration>
+        </execution>
+             
                     
+# Getting Started with Bootstrap-plus FitNesse Theme
+This is meant for people who want to develop for the Bootstrap-plus FitNesse Theme.
+
+## Running the code 
+
+1. Put the correct Bootstrap plus version in the pom.xml.
```

Comment:
Deze instructie is mij niet duidelijk. Als ik bootstrap+ uitcheck dan krijg ik toch al een correcte pom?

Reply:
De pom heeft inderdaad al de correcte versie nummer wanner je hem uitcheckt. Tekst is eruit gehaald.